matrix/lambdas/api/core.py

```diff
@@ -59,14 +63,16 @@ def post_matrix(body: dict):
 
     request_id = str(uuid.uuid4())
     RequestCache(request_id).initialize()
-
+    cloudwatch_handler.put_metric_data(
+        metric_name=MetricName.REQUEST,
+        metric_value=1
+    )
```

Comment:
Thoughts on coupling these with specific functions where appropriate? e.g. putting this one in `RequestCache.initialize`. Though it's true this doesn't make sense for all calls to `put_metric_data`, I think the coupling is cleaner and easier to maintain.